pkgs/by-name/de/deck/package.nix

```diff
@@ -36,4 +36,4 @@ buildGoModule rec {
     license = licenses.asl20;
     maintainers = with maintainers; [ liyangau ];
   };
-}
+}
```

Comment:
There should be an empty line. Does your editor read [.editorconfig](https://github.com/NixOS/nixpkgs/blob/master/.editorconfig)?